jsonschema-module-jackson/src/test/java/com/github/victools/jsonschema/module/jackson/JacksonHelperTest.java

```diff
@@ -0,0 +1,121 @@
+package com.github.victools.jsonschema.module.jackson;
+
+import com.fasterxml.jackson.annotation.JacksonAnnotationsInside;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.util.Optional;
+
+/**
+ * Unit test class dedicated to the validation of {@link JacksonHelper}.
+ *
+ * @author Antoine Malliarakis
```

Comment:
I'll rather include you in the `jsonschema-generator-parent/pom.xml` list of `contributors` and continue leaving out any `@author` annotations in individual files' javadocs.
Git blame still preserves that information anyway. 😉

Reply:
OMG. I missed that one ! :facepalm: All apologies will remove !

Reply:
Fixed in https://github.com/victools/jsonschema-generator/pull/487/commits/f128992e829da119f2167b2e472e6bf8e72aff60

Reply:
Nothing to apologize for 😄
Credit where credit is due. I'm just in favor of consistency and opted to collect the names of contributors in that `pom.xml` instead of spreading it all over the codebase.